app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less

```diff
@@ -648,6 +648,20 @@
     }
 }
 
+//
+//  Mobile @screen__xs
+//  _____________________________________________
+
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__xs) {
+    .product-info-main {
+        .product-info-price {
+            .price-box {
+                width: 1px;
+            }
+        }
+    }
+}
+
```

Comment:
Doesn't make sense. Looks like this:
<img width="344" alt="Screenshot 2019-12-12 at 20 31 02" src="https://user-images.githubusercontent.com/34317830/70742698-63deed80-1d1e-11ea-8113-a30cf694aa12.png">

Reply:
And I guess you want to achieve this:
![image](https://user-images.githubusercontent.com/34317830/70742744-86710680-1d1e-11ea-998d-e3259e90fc7b.png)